pkg/connector/vault.go

```diff
@@ -2,6 +2,11 @@ package connector
 
 import (
 	"context"
+	"errors"
+	"fmt"
+	"github.com/grpc-ecosystem/go-grpc-middleware/logging/zap/ctxzap"
+	"go.uber.org/zap"
+	"strings"
```

Comment:
If you move the strings import to line 7 (above the 3rd party dependencies), it should fix this linter error.

Also you should be able to run `make lint` to check for linter errors locally.